app/models/concerns/language_and_schedule_references.rb

```diff
@@ -5,6 +5,20 @@ module LanguageAndScheduleReferences
     has_many :language_skills, as: :languageable, dependent: :destroy
     accepts_nested_attributes_for :language_skills, allow_destroy: true
 
+    def first_language
+      return [] if language_skills.blank? || language_skills.native.blank?
+      return language_skills.native.german if language_skills.native.german.present?
+      return language_skills.native.limit(1) if language_skills.native.size.positive?
+    end
```

Comment:
I'm totally aware that this part isn't very elegant at all. But also with your solution, I'm struggling with the problem that I couldn't sort out a scope that relyably only delivered me one item, that isn't or is enclosed in an array.
So I got it crashing either because i was doing `LanguageSkill.first`, or becasue I was returning `[LanguageSkill]` and then the view crashed because of doing `[LanguageSkill].language_name`